content/docs/SUSHI/tips/_index.md

```diff
@@ -120,14 +120,15 @@ Logical: MyLM
 Id: MyLM
 Title: "My LM"
 Description: "This is mine"
-* important 1..1 SU boolean "Is this resource important"
+* id 1..1 SU string "Identifier for the logical model"
```

Comment:
Should this be the FHIR `id` type instead? It's a LM, so technically it can be _anything_. But since there is a FHIR primitive type called `id`, I think maybe we should use that to avoid any confusion.

Reply:
I totally agree. I had just forgotten that `id` was a primitive type, so I'll update that now.

Reply:
Added in [695c9f7](https://github.com/FSHSchool/site/pull/86/commits/695c9f7483e0b27ce8c0c24fe5fe71fd21e4d4c9).